tests/test_configs.py

```diff
@@ -1,29 +1,49 @@
 import hydra
+import pytest
 from hydra.core.hydra_config import HydraConfig
-from omegaconf import DictConfig
+from omegaconf import DictConfig, open_dict
 
+from tests.helpers.run_if import RunIf
 
+train_experiments = [
+    ["experiment=cls_modelnet_pointnet++"],
+    ["experiment=seg_s3dis1x1_pointnet++"],
+]
+
+
+@RunIf(openpoints=True)
+@pytest.mark.parametrize("cfg_train", train_experiments, indirect=["cfg_train"])
 def test_train_config(cfg_train: DictConfig):
-    assert cfg_train
-    assert cfg_train.datamodule
-    assert cfg_train.model
-    assert cfg_train.trainer
+    _test_config(cfg_train)
 
-    HydraConfig().set_config(cfg_train)
 
-    hydra.utils.instantiate(cfg_train.datamodule)
-    hydra.utils.instantiate(cfg_train.model)
-    hydra.utils.instantiate(cfg_train.trainer)
+eval_experiments = [
+    ["model=cls_pointnet++", "data=cls_modelnet2048", "ckpt_path=."],
+    ["model=seg_pointnet++", "data=seg_s3dis1x1", "ckpt_path=."],
```

Comment:
does ckpt_path need to be set?

Reply:
Yuh https://github.com/CCInc/3d-ml/blob/main/configs/eval.yaml